samples/star-wars-api/HttpHandlers.fs

```diff
@@ -175,68 +174,113 @@ module HttpHandlers =
                     return bytesRead > 0
             }
```

Comment:
maybe move it inside `isRequestIntrospection`?

Reply:
Attempting to do this a few different ways has given me an error like this every time:

![Screen Shot 2023-03-03 at 11 56 58 AM](https://user-images.githubusercontent.com/17415530/222814574-53f29e20-44c2-4966-b183-88dc31ad2de4.png)

I'm just going to leave it as-is.